suprb/optimizer/rule/ns/ns.py

```diff
@@ -25,9 +28,14 @@ class NoveltySearch(RuleGeneration):
             Iterations to evolve rules. Must be greater than zero.
         mu: int
             The amount of offspring from each population that get selected for the next generation.
-        lm_ratio: int
-            The ratio of lambda and mu. Each generation lambda=lm_ration*mu children will be generated but only mu will
-            survive.
+        lmbda: int
+            Each generation lambda children will be generated.
+        random_state : int, RandomState instance or None, default=None
+            Pass an int for reproducible results across multiple function calls.
+        n_jobs: int
+            The number of threads / processes the optimization uses. Currently not used for this optimizer.
+        n_elitists: int
+            The number of parents that get added to the population each generation
```

Comment:
Add here: For better performance this should be greater than the n_rules parameter of the optimize calls 
We should check this some time but conceptually that assumption is very sound